.travis.yml

```diff
@@ -7,7 +7,7 @@ rvm:
   - 1.9
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.2
```

Comment:
Rails 5.0.x has a minimum ruby version requirement of 2.2.2. Specifying just `2.2` to Travis resolves as version 2.2.0 and causes bundler to fail.